docs/technical-documentation/dev-process/Dev-flow_deploy-dev-env.md

```diff
@@ -0,0 +1,31 @@
+# Dev flow with deployment to dev environment
+
+```mermaid
+flowchart LR
+    subgraph local
+    D(Developer)
+    end
+    subgraph eclipse-tractusx
+        direction LR
+        D -- PR* to dev*--> PH(policy-hub**)
+        click PH "https://github.com/eclipse-tractusx/policy-hub"
+    end
+    subgraph Argo CD - sync to k8s cluster
+    PH -- auto-sync --> A(Argo CD dev)
+    click A "https://argo.dev.demo.catena-x.net"
```

Comment:
Please remove this link, as link to argo dev not sustainable and referencing consortia specific env

Reply:
done